Tests/BotTests/MergeService/Stubs/CommitState+Stub.swift

```diff
@@ -0,0 +1,35 @@
+@testable import Bot
+
+extension CommitState {
+    static func stub(states: [CommitState.State]) -> CommitState {
+        guard !states.isEmpty else {
+            return CommitState(state: .pending, statuses: [])
+        }
+
+        let combinedState: CommitState.State
```

Comment:
if we are stubbing commit state to behave like this, why do we need the same logic in the implementation at https://github.com/babylonhealth/Wall-E/pull/31/files#diff-39d0b4d1a105d5e23d5b7b2556fb51fdR431? Won't github commit status property follow the same rules? If not then why are we stubbing it like it does?

Reply:
can you add these details about `commit.status` to the doc comment of a `getRequiredChecksState` method?

Reply:
Just extracted it in a dedicated method with proper doc comment in a69dfbb03f5e169cc12dfea8f5eb989ed502f6b0 .

![image](https://user-images.githubusercontent.com/216089/67870893-d5c4f400-fb2f-11e9-97ed-3d50bba8cf36.png)